e2e/libs/recurringjob/rest.py

```
@@ -181,3 +185,26 @@ def cleanup(self, volume_names):
+            system_backup_list = filter_cr("longhorn.io", "v1beta2", "longhorn-system", "systembackups",
+                                           label_selector=f"recurring-job.longhorn.io/system-backup={job_name}")
+            try:
+                if len(system_backup_list['items']) == 0:
+                    continue
+
+                for item in system_backup_list['items']:
+                    state = item['status']['state']
+                    assert state == expected_state
+
+                return
+            except AssertionError as e:
```

Ruff (0.8.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)